week-5/C-promises/exercise.js

```diff
@@ -10,18 +10,28 @@
 
   Insert the result of promise1 into the #exercise1 element. Hint: .then()
 */
+
+var pEx1 = document.querySelector("#exercise1")
```

Comment:
This is not wrong, but personally I would have defined the `pEx1` variable inside the `.then` handler function. This is because we only really need to use the variable inside the `.then` handler function and nowhere else. If it's defined outside of this scope it's a bit unnecessary. We sometimes call this "polluting the scope".

Again, it's not wrong, just a question of style and that comes with experience.